toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -124,6 +126,30 @@ func validateConfig(baseConfigPath string, config *imagecustomizerapi.SystemConf
 		}
 	}
 
+	for i, script := range config.PostInstallScripts {
+		err = validateScript(baseConfigPath, &script)
+		if err != nil {
+			return fmt.Errorf("invalid PostInstallScripts item at index %d: %w", i, err)
+		}
+	}
+
+	for i, script := range config.FinalizeImageScripts {
+		err = validateScript(baseConfigPath, &script)
+		if err != nil {
+			return fmt.Errorf("invalid FinalizeImageScripts item at index %d: %w", i, err)
+		}
+	}
+
+	return nil
+}
+
+func validateScript(baseConfigPath string, script *imagecustomizerapi.Script) error {
```

Comment:
maybe also validate it has execution permission?

Reply:
Done.